obsidian.h

```diff
@@ -1,32 +1,91 @@
 #ifndef OBSIDIAN_H
 #define OBSIDIAN_H
 
+// Includes
+#include "unrealircd.h"
+#include "sqlite3.h"
+
 // Database files
-#define ACCOUNT_DB_PATH "../data/obsidian-account.db"
+#define OBSIDIAN_DB "../data/obsidian.db"
```

Comment:
This should be configurable somehow maybe not yet but later, where to store it.

Reply:
why?

Reply:
Because in a docker setup I want to configure exactly what needs to be stored and the path of a volume so that I only store what needs to be stored while the rest of the container is always re-created.

There might be also many non docker/container related reasons for why someone wants to decide where the persistence data should stay at.

Reply:
This path is actually ok and doesn't seem you can configure this anyway, so it is fine.